client/src/components/Tools.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
```

Comment:
Is this class a hard coded version of CollapsePanel?
Could you remove this class and add more passable props to CollapsePanel to achieve the same thing?

Reply:
Yes, it's hard coded at the moment for the ToolList. The class exists here to manage the expanded state of the panel and chose to put it here since it's managing its own state. I am hesitant to store the state of the panel in a different component but happy to talk about wats to improve this.

Reply:
Changed this to a simple named CollapsePanel wrapper.  If desired it can be pulled into the SignCampaign file sense that is the only place where it is used.